### DISCUSSION TITLE
Frontend

### DISCUSSION THREAD

**Discussion by BenzeneAlcohol** [NITK-KODE/cms-api#7] [General]
I can work on the frontend part on React, anyone else interested? We can use the Material UI design style for uniformity.

**Comment by BenzeneAlcohol** [comment]
I think we might need a separate repo for the frontend, what are your thoughts, guys?

**Reply by anasaijaz** [reply]
`Yea count mee in `

**Comment by priyansh32** [comment]
yeah surely we will need separate repo, it will be easier to host and collaborate
